llvm/include/llvm/Support/Error.h

```diff
@@ -1261,6 +1262,15 @@ inline Error createStringError(std::errc EC, char const *Fmt,
   return createStringError(std::make_error_code(EC), Fmt, Vals...);
 }
 
+template <typename... Ts>
+inline Error createStringErrorv(std::error_code EC, const char *Fmt,
```

Comment:
nit: I like the idea but it's really hard for me to tell this apart from the old function -- the only difference is the trailing lowercase `v` at the end. Could we either pick a different name or capitalize it consistently with the function (`V`)?

Reply:
I'd be fine with capitalizing the `v` at the end. I'm also alright with `createStringErrorVariadic` as well. If anyone feels particularly strongly about either of these, let me know. Otherwise I will likely choose the former.